### PR TITLE
feature/metrics cleanups 

### PR DESCRIPTION
**Pull request by purbon**
* cleanups debug variable used inside the pipeline.
* fixed small typo in the memory command.

### CODE REVIEW

**Comment by purbon** [comment]
@jsvd @ph thoughts?

File: logstash-core/lib/logstash/webserver.rb [hunk 1]
```diff
@@ -20,7 +20,8 @@ def initialize(logger, options={})
       http_port    = options[:http_port] || 9600
       @options     = {}
       @cli_options = options.merge({ :rackup => ::File.join(::File.dirname(__FILE__), "api", "init.ru"),
-                                     :binds => ["tcp://#{http_host}:#{http_port}"] })
+                                     :binds => ["tcp://#{http_host}:#{http_port}"],
+                                     :debug => logger.debug? })
```
Comment (jsvd) [inline logstash-core/lib/logstash/webserver.rb]: is this still necessary? isn't the logger passed to the webserver?

Comment (purbon) [inline logstash-core/lib/logstash/webserver.rb]: Is necessary to tell puma he is suposed to work on debug mode, only
possible throught cli options.

Comment (purbon) [inline logstash-core/lib/logstash/webserver.rb]: and to provide more context, no logger is not passed to any webserver here. This works just as interface to puma. Logger is also used inside the sinatra app, but this is another moving component.

**Comment by jsvd** [comment]
LGTM

**Pull request closed without merging**